### ISSUE TITLE
Add a function to return rows modified

### ISSUE THREAD

**Issue by RenathoAzevedo** [SqliteModernCpp/sqlite_modern_cpp#142]
Hi there, would be nice if you could add a function to return the count of database changes.
I added it by myself, but if you could add to the library:

in sqlite_modern_cpp/hdr/sqlite_modern_cpp.h
```
// line:459 - after this lines:
sqlite3_int64 last_insert_rowid() const {
	return sqlite3_last_insert_rowid(_db.get());
}
//add this:
sqlite3_int64 rows_modified() const {
    return sqlite3_changes(_db.get());
}
```
 thank you in advance!

**Comment by zauguin** [comment]
Thanks for the proposal. Can you create a [Pull request](https://help.github.com/articles/creating-a-pull-request-from-a-fork/) for this change?

**Comment by zauguin** [comment]
Merged in 9b53a7b2fd7f4c3130839320cc4f7148c0dea57c.
